study.md

```diff
@@ -20,21 +20,22 @@ material. Instead, digest what you've read and repeat it in your own voice.
 In your own words, describe the event loop and how it works.
 
 ```md
-<!-- your answer here -->
+The event loop is a programming tool that uses events in a program. It makes a request to a provider that then calls an event handler, depending on the nature of the request.
 ```
```

Comment:
That's not what the event loop is, I think you're thinking of event handlers https://www.youtube.com/watch?v=8aGhZQkoFbQ ← watch this